project_organization.md

```diff
@@ -0,0 +1,57 @@
+
+1. **Tracing tool maintainers:** "Instrumenting all software" is an unreasonable goal for any one particular tracing or monitoring project or vendor. OpenTracing essentially amortizes this work.
+2. **Software developers who build and deploy applications:** These developers want to use whatever tracing and observability tools that work best within their organization's infrastructure, and they want to choose those tools independent of whatever third-party (open-source) software packages they happen to have built around.
+3. **Software developers who contribute to widely-used software:** Inasmuch as this software needs to exist within a microservices deployment, it must integrate with whatever tracing tool(s) its diverse users already depend on.
+
+OpenTracing's project organization makes room for each of these constituencies.
+
+## OpenTracing Specification Council (OTSC)
+
+The authors of any tracing or observability tool that supports OpenTracing are **always** invited to contribute to discussions about the future of OpenTracing as a standard. OpenTracing endeavors to maintain a list of such authors and will occasionally solicit feedback from that group.
+
+A subset of these tracing system authors are part of the **OpenTracing Specification Council** (OTSC). Said council members have write access across the [github.com/opentracing](https://github.com/opentracing) and [github.com/opentracing-contrib](https://github.com/opentracing-contrib) organizations (though other individuals may also have write access to specific repos within those organizations). Each OTSC member represents a tracing system which has substantial institutional support (and thus longevity). The current OpenTracing Specification Council is as follows (in alphabetical order):
+
+- Bas van Beek ([@basvanbeek](https://github.com/basvanbeek)): Zipkin
+- Ben Sigelman ([@bensigelman](https://github.com/bensigelman)): LightStep
+- Pavol Loffay ([@pavolloffay](https://github.com/pavolloffay)): Hawkular
+- Yuri Shkuro ([@yurishkuro](https://github.com/yurishkuro)): Jaeger
+
+OTSC members have the following primary responsibilities:
+
+- Define priorities for OpenTracing as a project and standard
+- Accurately represent the concerns and needs of downstream tracing and observability tools
+- Strategize around possible third-party package integrations
+- Weigh in and offer tie-breaking votes for issues where consensus among the larger community has proven elusive
+
+## OpenTracing Industrial Advisory Board (OTIAB)
+
+OpenTracing has an **Industrial Advisory Board** (OTIAB) that comprises engineers with detailed knowledge of tracing technology at companies of different scales and software maturity. The OTSC members engage with the OTIAB on a periodic basis to gather feedback about successes, challenges, and open-source packages which would benefit from greater OpenTracing support.
+
```

Comment:
I added a note about this. There is an initial list, but I want to make sure we're bringing in people beyond my personal network before announcing.